smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/json/MemberShapeDecodeGenerator.kt

```diff
@@ -153,15 +154,15 @@ abstract class MemberShapeDecodeGenerator(
             writer.write("var \$L:\$T = nil", decodedMemberName, originalSymbol)
             writer.openBlock("if let \$L = \$L {", "}", listContainerName, listContainerName) {
                 writer.write("\$L = \$N()", decodedMemberName, originalSymbol)
-                renderDecodeListTarget(nestedTarget, decodedMemberName, listContainerName, insertMethod, topLevelMember, shape.isSetShape, level)
+                renderDecodeListTarget(nestedTarget, decodedMemberName, listContainerName, insertMethod, topLevelMember, shape, level)
             }
             renderAssigningDecodedMember(topLevelMember, decodedMemberName)
         } else {
             writer.openBlock("if let \$L = \$L {", "}", memberName, memberName) {
                 val previousDecodedMemberName = "${memberName.removeSurroundingBackticks()}Decoded${level - 1}"
-                val symbolName = determineSymbolForShape(shape, isSparse)
+                val symbolName = determineSymbolForShape(shape, false)
```

Comment:
**Question**: Is `isSparse` used anywhere else in this method?

Reply:
good catch, no reference for `isSparse`, cleaned.